roles/common/tasks/ntp.yaml

```diff
@@ -1,27 +1,28 @@
 ---
 - name: 'NTP. Install Package'
   yum:
-    name: "{{ ntp_package }}"
+    name: "{{ item }}"
     state: present
     update_cache: true
+  with_items: "{{ package_list }}"
 
 - name: 'NTP. Configure Service'
   template:
     src: ntp.conf.j2
-    dest: "{{ ntp_configfile }}"
-    owner: "{{ ntp_configfile_user }}"
-    group: "{{ ntp_configfile_group }}"
+    dest: /etc/ntp.conf
+    owner: root
+    group: root
     mode: 0644
     backup: true
   notify:
     - restart ntp
 
 - name: 'NTP. Start Service'
   service:
-    name: "{{ ntp_service }}"
+    name: ntpd
     state: started
     enabled: true
 
-- name: 'Timezone. Set to Europe/Kiev'
+- name: 'Setting up default timezone'
```

Comment:
But if I redefine the variable, then it will no longer be a default. It is better to set `'Setting up {{ ntp_timezone }} timezone'`

Reply:
thanks. good solution.